www/includes/easyparliament/templates/html/search/form.php

```diff
@@ -0,0 +1,92 @@
+        <form>
+            <div class="search-page__section search-page__section--search">
+                <div class="search-page__section__primary">
+                    <p class="search-page-main-inputs">
+                        <input type="text" name="q" value="<?= $search_keyword ?>" class="form-control">
+                        <button type="submit" class="button">Search</button>
+                    </p>
+                    <p>
+                        <ul class="search-result-display-options">
+                        <li><a href="#options" class="search-options-toggle js-toggle-search-options">Advanced search</a></li>
+                        <?php if ( $is_adv ) { ?>
+                            <?= $search_from ? "<li>From: $search_from</li>" : '' ?>
```

Comment:
I think these are all HTML injections, unless I've missed the escaping somewhere.